plugin/Form/FormAddressCreator.php

```diff
@@ -82,6 +82,13 @@ public function createOrCompleteAddress(
 		if ( key_exists( 'newsletter', $requestParams ) ) {
 			unset( $requestParams['newsletter'] );
 		}
+		if (!empty($estateId)) {
+			$userName = $this->getSupervisorUsernameByEstateId($estateId);
+			if (!empty($userName)) {
```

Comment:
In the past we got minor bugs with using empty() function because "0" and 0 is also empty true. It is unlikely but what happens if the ID or username is 0?

Do you think using isset could be better or could causes other errors?

Edit:
Oh I see getSupervisorUsernameByEstateId returns maybe an empty string. So I guess we need empty() here.

Reply:
I'll check and fixed it asap